src/Security/Security.php

```diff
@@ -1150,7 +1150,7 @@ public static function has_default_admin()
     /**
      * Get default admin username
      *
-     * @deprecated 4.0.0:5.0.0 Use DefaultAdminService::getDefaultAdminUsername()
+     * @deprecated 4.0.0 Use DefaultAdminService::getDefaultAdminUsername()
```

Comment:
Update the notice message and version as well.
```suggestion
     * @deprecated 4.0.0 Use DefaultAdminService::getDefaultAdminUsername() instead
```